autonomi/src/python.rs

```diff
@@ -1067,6 +1298,195 @@ impl PyClient {
             }
         })
     }
+
+    /// Verify a pointer
+    #[staticmethod]
+    fn pointer_verify(pointer: &PyPointer) -> PyResult<()> {
+        Client::pointer_verify(&pointer.inner)
+            .map_err(|e| PyRuntimeError::new_err(format!("Failed to verify pointer: {e}")))
+    }
+
+    fn get_raw_quotes<'a>(
+        &self,
+        py: Python<'a>,
+        data_type: PyDataTypes,
+        content_addrs: Vec<(PyXorName, usize)>,
+    ) -> PyResult<Bound<'a, PyAny>> {
+        let data_type: DataTypes = data_type.into();
+        let client = self.inner.clone();
+        let content_addrs_iter = content_addrs
+            .into_iter()
+            .map(|(xor_name, size)| (xor_name.inner, size));
+
+        future_into_py(py, async move {
+            let results = client.get_raw_quotes(data_type, content_addrs_iter).await;
+
+            let py_results: Vec<_> = results
+                .into_iter()
+                .map(|result| match result {
+                    Ok((xor_name, quotes)) => {
+                        let xor_name_str = xor_name.to_string();
+                        let quote_data: Vec<_> = quotes
+                            .into_iter()
+                            .map(|(peer_id, quote)| {
+                                (
+                                    peer_id.to_string(),
+                                    quote.content.to_string(),
+                                    format!("{:?}", quote.timestamp),
+                                    format!("{:?}", quote.quoting_metrics),
+                                    format!("0x{}", hex::encode(quote.rewards_address.as_slice())),
+                                    hex::encode(&quote.pub_key),
+                                    hex::encode(&quote.signature),
```

Comment:
Would be better to return the actual quotes (wrapped) not stringified to we can use it in other functions here